components/segment-bridge/README.md

```diff
@@ -0,0 +1,7 @@
+# SEGMENT-BRIDGE
+
+Component used to allow reading resources from the staging member cluster
```

Comment:
```suggestion
Component used to allow reading resources from the host and member clusters.
```

I see you've got duplicate changes in https://github.com/redhat-appstudio/infra-deployments/pull/2054/files#diff-442738682d1dc0a832b81da44904ca09c3a5c47b534ffcd462f00ac81226be80R3. You might as well make one version accurate.

Reply:
I guess you're right for this specific line, otherwise I preferred each change to be atomic, as rebasing will need to happen either way...